### PR TITLE
Add responsive and Buttons

### PR DESCRIPTION
**Pull request by eLefevr**
## Responsive

Add bootstrap & jquery min files (useful during the RoboCup) and create a js file for the script in the html file.

The web page is now responsive and can be displayed on a tablet screen. 

The displayed text at the bottom of the page do not get out of the div and is displayed in a more convenient way (last received message first) and I also add a prefix when displaying the message (as it's done in the commented examples in the html file). 

I also add an area to put the name of the current test in the sidebar. 

## Add buttons in order to use Vizbox as an alternative HRI

I add two buttons, `next step` and `stop` which publish a message in a ros topic called `next_step` ( `rostopic echo /next_step` to see the messages).

Messages published can be modified in server.py by modifying vars CMD_BTN1 and CMD_BTN2.

### CODE REVIEW

**Comment by LoyVanBeek** [comment]
Thnx a lot! I'll take a proper look this evening, very curious.

File: templates/challenge.html [hunk 1]
```diff
@@ -2,84 +2,52 @@
...
+        <div class="content center">
+            {{visualization}}
+            <br>
+            <img    id="visualization_img"
+                    src="https://www.w3schools.com/html/html5.gif">
+        </div>
 
-                        console.log("Got story: '"+json+"'")
+        <div class="sidebar">
+            <h2>Epreuve</h2>
```
Comment (LoyVanBeek) [inline templates/challenge.html]: Does this translate to 'Event' ? Please put this in English

File: static/scripts/script.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    switch(json.label)
+    {
+        case "operator_text":
+        case "robot_text":
+            var subtitles = document.getElementById("subtitles");
+            var txt = document.createElement("li");
+            txt.appendChild(document.createTextNode(json.text));
+            txt.setAttribute("class", json.label + " subtitle-line");
+            console.log(json.label) // added line
+            subtitles.insertAdjacentElement('afterbegin', txt);
```
Comment (LoyVanBeek) [inline static/scripts/script.js:15]: The typical chat log appends at the bottom. 
Also clearing the current dummy-content the first time text is added would be nice too.

**Review by LoyVanBeek — CHANGES_REQUESTED** [review]
Code is getting in much better shape this way, thnx! 

The 'Next step' and 'Stop' buttons aren't working for me though, There's nothing published to `/next_step` when I click it. 
Or I misunderstood, in which case the README could also use an hint on this.

**Comment by LoyVanBeek** [comment]
I have some work to do myself after this is merged: add a form to input commands for the robot at the very bottom of the page (chat-application style)

File: static/scripts/script.js (hunk 2 above)
Comment (eLefevr) [reply static/scripts/script.js:15]: I don't understand why you want to clear the dummy-content, for me it's just an example to see how messages will appear, there is no reason to keep it displayed during the cup. 
If it is mandatory to you, I can keep the dummy messages and remove them when receiving new text.

Comment (LoyVanBeek) [reply static/scripts/script.js:15]: No dummy text is fine too

**Comment by eLefevr** [comment]
I don't understand which kind of form you want to do. 
I don't think that write text on this interface will be the more convenient since VizBox will not only be displayed on computers. 
Maybe some dropdowns can do the job you want

**Comment by LoyVanBeek** [comment]
The form I'd like to maybe add would be just a textbox and a submit button, where you enter+send what you want to say to the robot, to add the VizBox as an easy way to say something to the robot. 

Hiding that by default and opening it with a button would be handy, because not everyone will need it.

**Review by LoyVanBeek — APPROVED** [review]

**Pull request merged**